### DISCUSSION TITLE
Interested in Contribution

### DISCUSSION THREAD

**Discussion by ARYANGAUATM001** [SeaQL/summer-of-code#18] [General]
I am final year engineering student from India. I am proficient in  SQL and rust can help in contributing to the project. I write clean SQL queries and can help with documentation too . I would like to contribute to the project.

**Comment by billy1624** [comment]
Hey @ARYANGAUTAM001, we're always here to welcome new contributors! https://www.sea-ql.org/blog/2023-01-01-call-for-contributors-n-reviewers/

Or, you mean the application of our upcoming summer internship in 2023? If so, please send your cover letter, along with your curriculum vitae and academic transcript, to hello@sea-ql.org via email.
